docs/tests.md

```diff
@@ -27,7 +27,6 @@ Make sure that you include at least these two tests:
 
     Use the `RunTestUpgrade()` test helper to help determine compatibility with earlier module versions.
 
-    - If you must implement a breaking change to the module in a PR, include a Conventional Commit footer in one of the commits with the text `BREAKING CHANGE:` at the beginning of the footer. For more information, see the [Conventional Commits cheat sheet](https://cheatography.com/albelop/cheat-sheets/conventional-commits/).
     - If the change breaks example code but not module code, include `SKIP UPGRADE TEST` in the commit message. Make sure to describe why you are breaking the example in the PR description.
 
   ?> **Tip**: The upgrade test is disabled by default in `pr_test.go` in the module template because you can't run an upgrade test until the initial module code is merged to the main branch. After the initial merge, create a pull request to enable the upgrade test by commenting out the line that starts with `t.Skip`.
```

Comment:
This is no longer a requirement as the line that starts with `t.Skip` has been removed from the module template (too many people were not removing it, and tests were being skipped). Perhaps we should mention that `SKIP UPGRADE TEST` may need to be used in one of the commit message for a modules first PR since there won't be any pre existing module code to do an upgrade test with.

Reply:
how about this?

<img alt="image" src="https://github.com/terraform-ibm-modules/documentation/assets/8194208/8b5b2b2b-e41c-4e76-acdf-d2c5ed196f2c">